MQTicketService/Dockerfile

```diff
@@ -47,10 +46,11 @@ USER 0
 #
 #
 RUN if [ "$platformArch" == "amd64" ]; then \
-    curl -Lo OpenJDK11U-jre_x64_linux_hotspot_11.0.13_8.tar.gz https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jre_x64_linux_hotspot_11.0.13_8.tar.gz && \
-    ls -l OpenJDK11U-jre_x64_linux_hotspot_11.0.13_8.tar.gz && file OpenJDK11U-jre_x64_linux_hotspot_11.0.13_8.tar.gz && \
-    tar -xvzf OpenJDK11U-jre_x64_linux_hotspot_11.0.13_8.tar.gz; \
+    curl -Lo OpenJDK11Ux64.tar.gz https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jre_x64_linux_hotspot_11.0.13_8.tar.gz && \
+    ls -l OpenJDK11Ux64.tar.gz && file OpenJDK11Ux64.tar.gz && \
+    tar -xvzf OpenJDK11Ux64.tar.gz; \
 	fi
+# 
 #
 RUN if [ "$platformArch" == "arm64" ]; then \
```

Comment:
The two if statements deal with amd64 and arm64, but it's not clear what should happen if the value is set to something else e.g., 'typo64'. Ideally we should check for that and fail. One option would be to pre-check and make sure the two `if` branches are safe or add an if, elif and else.